srcpkgs/tbsm/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'tbsm'
+pkgname=tbsm
+version=0.5
+revision=1
+depends="bash"
+short_desc="Pure bash session or application launcher"
+maintainer="rc-05 <rc23@email.it>"
+license="GPL-2.0-or-later"
+homepage="https://loh-tar.github.io/tbsm/"
+distfiles="https://github.com/loh-tar/${pkgname}/archive/v${version}.tar.gz"
+checksum="1e3bfaaad99cf757142d1362f4c1eb357a04dacd45e024f7422902d6c4126a17"
+conf_files="/etc/tbsm.conf"
+
+do_install() {
+	vbin src/tbsm
+	vconf src/tbsm.conf
+
+	vmkdir etc/xdg/tbsm/themes
+	for theme in themes/*; do
+		vinstall $theme 644 etc/xdg/tbsm/themes
+	done
+
+	for doc in doc/*; do
+		vdoc $doc
```

Comment:
ditto

Reply:
Documentation here is distributed as text files.